### PR TITLE
Corrections to SecurityProtocolType

### PR DESCRIPTION
**Pull request by stevewhims**
# Title

## Summary

1. The example needed deleting: SslStream doesn’t even use SecurityProtcolType,
2. The Remarks section was wrong. Fixed.
3. The value for SystemDefault was clunky. The link to SChannel wasn’t very useful (SCHANNEL is an enormous surface area). Fixed.

### CODE REVIEW

File: xml/System.Net/SecurityProtocolType.xml
Comment (davidsh) [inline xml/System.Net/SecurityProtocolType.xml]: "you should use this field".  It is not a "field".  This is an enumeration value from the `SecurityProtocolType` enumeration.

So, that phrase would be better as:

"you should select this enum value"

**Review by davidsh — CHANGES_REQUESTED** [review]
Left some comments.

File: xml/System.Net/SecurityProtocolType.xml
Comment (stevewhims) [reply xml/System.Net/SecurityProtocolType.xml]: docs.microsoft.com calls them "fields".

Comment (davidsh) [reply xml/System.Net/SecurityProtocolType.xml]: That should be changed then.  "Fields" are within a class object.  They usually aren't something that is exposed publicly.

"Properties" on an object are something that can be "get" or "set".

But this documentation above is simply describing the different values from an enumeration type (SecurityProtocolType). In that case, "fields" does not seem right.

@terrajobst Can you comment on the proper .NET terminology for these things?

Comment (terrajobst) [reply xml/System.Net/SecurityProtocolType.xml]: Hey @mairaw. How are we referring to members of enums? Are we calling them fields?

@davidsh technically, enums are encoded in metadata very similar to classes/structs and their member are just public const fields. So either terminology would work for me.

File: xml/System.Net/SecurityProtocolType.xml
Comment (mairaw) [inline xml/System.Net/SecurityProtocolType.xml]: Instead of permissible, we should use something a bit easier maybe? Like allowed or acceptable?

File: xml/System.Net/SecurityProtocolType.xml
Comment (mairaw) [reply xml/System.Net/SecurityProtocolType.xml]: We do refer to them as fields but I think in this context I prefer value. 

However, this guidance is very light. Probably something more direct probably works better. 
"Unless your app has a specific reason not to, use this value."

File: xml/System.Net/SecurityProtocolType.xml
Comment (stevewhims) [reply xml/System.Net/SecurityProtocolType.xml]: Sure, that word was there before, and not part of my change. I'll change it, though.

File: xml/System.Net/SecurityProtocolType.xml
Comment (stevewhims) [reply xml/System.Net/SecurityProtocolType.xml]: I'm happy to change the word to "value" if that's the consensus, but it makes no sense to use a different term from the title that's right above it (and is on every Enum topic). I was hoping that the decision would be to leave it consistent but spin off another thread (probably private) to discuss whether or not to comprehensively lose the internals-seeming term "field" and move (consistently) to more usual language. But as I say I'll change it here (to be inconsistent) for the sake of moving on.

Comment (rpetrusha) [reply xml/System.Net/SecurityProtocolType.xml]: docs.microsoft.com lists enum members as fields, though MSDN simply described them as members; AFAIK, the change was simply due to an MSDN customization that wasn't ported to docs. They are fields, but we've generally not referred to them as fields, since that's an implementation detail, and most developers don't think of them as fields. So I'd say the guidance is to refer to a member as an enumeration member (which is generally what I do) or an enumeration value. I'd avoid field.
Here, since there's no question that the thing being documented is the enumeration member, I'd agree with @mairaw that the sentence best reads, "Unless your app has a specific reason not to, use this value."

File: xml/System.Net/SecurityProtocolType.xml
Comment (mairaw) [reply xml/System.Net/SecurityProtocolType.xml]: I know @stevewhims. But since you changed the sentence, it caught my attention. 😄 Thanks for changing.

File: xml/System.Net/SecurityProtocolType.xml
Comment (stevewhims) [reply xml/System.Net/SecurityProtocolType.xml]: I would say that that heading needs to be changed, then. That heading essentially says "here are some fields", but writers are being asked to call them something else *right below that heading*.

**Comment by stevewhims** [comment]
The PR is still showing "Changes requested", but I've made all the changes. The "see review" link is interactive but unresponsive (does nothing when you click it). Any ideas how I can get this PR merged?

**Comment by stevewhims** [comment]
I hesitate to click "Dismiss review". Is that what I'm meant to do here?

**Comment by BillWagner** [comment]
@stevewhims Since you've made the requested changes, @stevewhims needs to approve the changes. He should have a link that says "approve changes".

**Comment by mairaw** [comment]
@davidsh was the one who requested the changes so he should approve and you should be able to merge

**Comment by stevewhims** [comment]
Ok, thanks! :)

File: xml/System.Net/SecurityProtocolType.xml
Comment (davidsh) [inline xml/System.Net/SecurityProtocolType.xml]: > which is used by System.Net.Http.HttpClient

This is not actually correct.  

The real text should say something like:
"The ServicePointManager.SecurityProtocolType is used by the various HTTP APIs in .NET Framework which include System.Net.WebClient, System.Net.HttpWebRequest and System.Net.Http.HttpClient. ...

**Comment by davidsh** [comment]
>@davidsh was the one who requested the changes so he should approve and you should be able to merge

I added more comments for changes needed.

**Comment by davidsh** [comment]
And actually in thinking about this more.....this property also controls the default TLS protocols used for System.Net.Mail.SmtpClient (when using TLS).

**Comment by stevewhims** [comment]
Ok, cool. I've actioned the last 2 comments @davish: please check whether that's correct now. Btw, "this property" is actually "this enum". But if ServicePointManager.SecurityProtocol is the only property of this type then that distinction is real, but not very important.

**Review by davidsh — APPROVED** [review]

**Pull request merged**
